appserver/tests/cts_smoke/test_dir.properties

```diff
@@ -0,0 +1,8 @@
+servlet-api-servlet=servlet/api/javax_servlet
+servlet-api-servlet-http=servlet/api/javax_servlet_http
+servlet-compat=servlet/compat
```

Comment:
How does it handle the scenario if a new folder is added? Can one of the test id can cover "all remaning folders"